lib/SILOptimizer/Transforms/PruneVTables.cpp

```diff
@@ -98,6 +98,8 @@ class PruneVTables : public SILModuleTransform {
   }
   
   void run() override {
+    return;
```

Comment:
We tend to use comment markers to indicate changes on the tensorflow branch.  Would you mind adding that?

```suggestion
    // SWIFT_ENABLE_TENSORFLOW
    return;
    // SWIFT_ENABLE_TENSORFLOW END
```

Reply:
Are those used by anything automated or are they for human consumption?

Reply:
The markers are just for human consumption, so it's easier to identify `tensorflow` branch specific changes at a glance.